config/example/config.toml

```diff
@@ -646,7 +646,7 @@ redis_connstring = "redis://127.0.0.1/"
 # This section contains configuration for all space ruins and lava ruins
 
 # Whether to load the lavaland Z-level.
-enable_lavaland = true
+enable_lavaland = false
```

Comment:
revert this change

Reply:
For reference, if you copy the config and put it outside of the `/example` folder, it gets gitignored and you don't see these changes pop up

Reply:
Ah, I did that because I thought that the folder was supposed to be updated as new maps are added, as I saw that the last PR that added a ruin (the drake skiff) also did that. Reverting.